The build is failing, but I don't think that's due to my change:

```
3.02s$ npm install

npm ERR! Linux 3.13.0-115-generic

npm ERR! argv "/home/travis/.nvm/versions/node/v6.11.0/bin/node" "/home/travis/.nvm/versions/node/v6.11.0/bin/npm" "install"

npm ERR! node v6.11.0

npm ERR! npm  v3.10.10

npm ERR! code ETARGET

npm ERR! notarget No compatible version found: gulp-postcss@^6.4.1
```

--- a/docs/pages/visibility.md
+++ b/docs/pages/visibility.md
@@ -120,7 +120,7 @@ To hide text from assistive technology, while still keeping it visible, add the 
 
 If your site has a lot of navigation, a screen reader will have to read through the entire navigation to get to your site's content. To remedy this, you can add a *skip link* at the very top of your page, which will send the user farther down the page, past the navigation when clicked on.
 
-Use the class `.show-on-focus` to hide an element, except when it has focus. Adding tabindex="0" to the target element makes if focusable.
+Use the class `.show-on-focus` to hide an element, except when it has focus. Adding `tabindex="-1"` to the target element makes it focusable. (Or set it to `0` if the user should be able to tab to that element as well. See also [the MDN docs on `tabindex`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex).)
 
 ```html_example
 <p><a class="show-on-focus" href="#mainContent">Skip to Content</a></p>

